id.js

```diff
@@ -67,22 +67,17 @@ module.exports = function(opts) {
     get: function() {
       if ( ! this.hasOwnProperty(key__) ) {
         try {
+          var existingLocalId = this[key__];
           Object.defineProperty(this, key__, {
             value: nextId(),
             enumerable: false,
           });
+          if (existingLocalId === this[key__]) return getPairedId(this);
```

Comment:
nit: moar space! `_` = space; we want `if_(_predicate_)`. Same as elsewhere in this file.

(sorry that style isn't the same across all related repos).